src/hashtable.h

```diff
@@ -31,6 +31,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <unistd.h>
+#include <stdbool.h>
```

Comment:
Many hashtable functions return 1 or 0 as an int. This was done before we started allowing bool. We should probably change them all to bool. (A separate PR at some point.)